src/common/cockpitpipe.c

```diff
@@ -1116,11 +1116,18 @@ cockpit_pipe_connect (const gchar *name,
   else
     {
       if (!g_unix_set_fd_nonblocking (sock, TRUE, NULL))
-        g_return_val_if_reached (NULL);
+        {
+          close (sock);
+          g_return_val_if_reached (NULL);
```

Comment:
Any idea how to do that?

Would you prefer that I just remove this and tell coverity to ignore?